### PR TITLE
Use central wavelength for AGS thus able to calculate guide star for …

### PR DESCRIPTION
**Pull request by cquiroz**
…calibrations

### CODE REVIEW

**Comment by cquiroz** [comment]
@toddburnside we may want to verify we are using the same on the odb ags calculation

File: explore/src/main/scala/explore/tabs/ObsTabTiles.scala [hunk 1]
```diff
@@ -146,6 +148,19 @@ object ObsTabTiles:
     acquisition: Option[NonEmptyList[Offset]]
   )
 
+  // TODO Move to core
+  extension (om: ObservingMode)
+    def centralWavelength: Option[Wavelength] =
```
Comment (rpiaggio) [inline explore/src/main/scala/explore/tabs/ObsTabTiles.scala:153]: We have a `CentralWavelength` type alias, do we want to return that instead?

Comment (cquiroz) [reply explore/src/main/scala/explore/tabs/ObsTabTiles.scala:153]: Yeah, I think that would be better, I'll change it

**Review by rpiaggio — APPROVED** [review]
LGTM

**Pull request merged**